src/aura/STG_CMP_Affl/STG_CMP_AfflHelper.js

```diff
@@ -94,7 +94,14 @@
                 
                 // Re-Disable Add Setting button after successful completion
                 component.find("newAfflMappingBtn").set("v.disabled", true);
+                
+                // Re-Hide error message
+                component.set("v.showAfflError", false);
             } else if(response.getState() === "ERROR") {
+                var errors = JSON.stringify(response.getError());
+                if (errors.includes('There is already an item in this list with the name')) {                
```

Comment:
This error message is not going to change in a different language, is it? I don't think it would, but we should check, just in case...

Reply:
Good catch, looks like it is changing. I made a minor edit in the IF condition, so that it works in other languages too. Can you let me know your thoughts on the new IF? Thanks.

Reply:
<img width="444" alt="Screen Shot 2019-11-03 at 5 47 19 PM" src="https://user-images.githubusercontent.com/49822508/68096125-92d28b80-fe63-11e9-9348-6acdb81d97f1.png">